pkg/registry/cluster/storage/proxy.go

```diff
@@ -58,3 +58,8 @@ func (r *ProxyREST) Connect(ctx context.Context, id string, options runtime.Obje
 	}
 	return proxy.ConnectCluster(ctx, cluster, proxyOpts.Path, secretGetter, responder)
 }
+
+// Destroy cleans up its resources on shutdown.
+// Not implemented.
+func (r *ProxyREST) Destroy() {
```

Comment:
// Given that underlying store is shared with REST,
// we don't destroy it here explicitly.